scenario.py

```diff
@@ -20,10 +20,11 @@
 class Scenario:
     def __init__(self, params, experiment_path):
 
+        # Identify and get a dataset for running experiments
         self.dataset_name = "MNIST"
         (x_train, y_train), (x_test, y_test) = mnist.load_data()
-        # The train set has to be split into a train set and a validation set for early stopping (called 'esval' below)
-        self.x_train, self.x_esval, self.y_train, self.y_esval = train_test_split(
+        # The train set has to be split into a train set and a validation set for early stopping
+        self.x_train, self.x_valearlystop, self.y_train, self.y_valearlystop = train_test_split(
```

Comment:
`x_val` and `y_val` are the names traditionnaly used for validation datasets :)

Reply:
Yes, I think we need to have a clear and distinct name for the early stopping validation set, to make sure at the local node level there is no confusion with the actual validation set.